content/influxdb/v2.0/query-data/influxql.md

```diff
@@ -43,13 +43,60 @@ By default, the `/query` compatibility endpoint returns results in **JSON**.
 To return results as **CSV**, include the `Accept: application/csv` header.
 
 ## Database and retention policy mapping
+
 InfluxDB 2.0 combines the 1.x concept of [databases](/influxdb/v1.8/concepts/glossary/#database)
 and [retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp)
 into [buckets](/influxdb/v2.0/reference/glossary/#bucket).
+
+InfluxQL looks for a database and retention policy (elements in the FROM clause) to query from.
+To query a 2.0 bucket in InfluxQL, you must first map the bucket to a database and retention policy.
+
 To support InfluxDB 1.x query and write patterns in InfluxDB 2.0, databases and retention
 policies are mapped to buckets using the **database and retention policy (DBRP) mapping service**.
 _See [DBRP mapping](/influxdb/v2.0/reference/api/influxdb-1x/dbrp/) for more information._
 
+{{% note %}}
+If you use the 1.x compatibility API to write data into InfluxDB 2.0, the database and retention policy included as query parameters were mapped to a bucket. For more detail, see [Database and retention policy mapping](/influxdb/v2.0/reference/api/influxdb-1x/dbrp/).
+{{% /note %}}
+
+{{% note %}}
+The mapping is automatically created only if you use the [1.x `/write` compatibility endpoint](/influxdb/v2.0/reference/api/influxdb-1x/write/). If you used the 2.x write API and want to access that data using InfluxQL, you'll need to create a mapping. To review existing mappings, use the [`GET DBRPs` CURL command](/influxdb/v2.0/api/#operation/GetDBRPs).
```

Comment:
because the CURL command isn't shown in the API docs, lets include a CURL eg.:

```
curl -XPOST https://cloud2.influxdata.com/api/v2/dbrps \
  -H "Authorization: Token YourAuthToken" \
  -H 'Content-type: application/json' \
  -d '{
        "bucketID": "12ab34cd56ef",
         "db": "example-db",
         "orgID": "example-org"
         "rp": "example-rp",
     
      }'
```
Noting the orgID is the only required parameter. If folks want to search for specific db, rp, bucket, they can include that parameter